rmw_fastrtps_shared_cpp/src/rmw_take.cpp

```diff
@@ -336,24 +321,7 @@ _take_serialized_message(
   eprosima::fastdds::dds::SampleInfoSeq info_seq{1};
 
   while (ReturnCode_t::RETCODE_OK == info->data_reader_->take(data_values, info_seq, 1)) {
```

Comment:
Where are the loaned samples returned?

Reply:
The copy is already done when the sample is deserialized. Also, `StackAllocatedSequence` does not need to be returned. When leaving the scope the loan is directly freed.